setup.cfg

```diff
@@ -11,6 +11,10 @@ replace = version="{new_version}"
 search = __version__ = "{current_version}"
 replace = __version__ = "{new_version}"
 
+[bumpversion:file:CITATION.cff]
+search = version: {current_version}
+replace = version: {new_version}
+
 [bdist_wheel]
```

Comment:
Are you sure you want this? This will increment the `cff-version` when the toolbox updates. Isn't that version number to do with the cff format rather than our toolbox version.

Reply:
Agree, resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/228/commits/89b0923438c1fa9bf3c5d1832f7617a7062473ec